css_tasks/task9/scss/_jumbotron.scss

```diff
@@ -0,0 +1,206 @@
+@import "variables";
+@import "mixins";
+
+.jumbotron {
+  position: relative;
+  &__image {
+    width: 100%;
+    max-height: 40rem;
+    opacity: 0.82;
+    background: #000;
+  }
+  &__content {
+    position: absolute;
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    grid-column-gap: 2rem;
+    top: 13.16%;
+    .jumbotron-content__left-column {
+      @include flexColumn;
```

Comment:
nothing changed without this styles, it is possible that even class not really needed